include/Dialect/Polynomial/IR/PolynomialAttributes.td

```diff
@@ -39,18 +39,26 @@ def Ring_Attr : Polynomial_Attr<"Ring", "ring"> {
     An attribute specifying a polynomial quotient ring with integer
     coefficients, $\mathbb{Z}/n\mathbb{Z}[x] / (p(x))$.
 
+    `cwidth` is the bit width of the coefficient storage type,
     `cmod` is the coefficient modulus $n$, and `ideal` is the ring ideal
     $(p(x))$. Because all ideals in a single-variable polynomial ring are
     principal, the ideal is defined by a single polynomial.
 
       #ring = #polynomial.ring<cmod=1234, ideal=#polynomial.polynomial<x**1024 + 1>>
```

Comment:
nit: add `cwidth` here since it's required

Reply:
This is actually a bit of a trick. The custom parser/printer for this attribute doesn't include the `cwidth`, and the builder infers it from the `cmod` attribute. So it's not needed in the docs anywhere, really. I will just remove that line 42.